fhir/group.js

```diff
@@ -6,11 +6,11 @@ const lib    = require("../lib");
 
 const SERVER_START_TIME = moment().format("YYYY-MM-DD HH:mm:ss");
 
-function resourceCreator(multiplier) {
+function resourceCreator(req, multiplier) {
```

Comment:
It seems strange to me for a 'resourceCreator' method require a 'request' in order to work, when all it is getting out of the request is a base URL.  I'd have the signature be
```
function resourceCreator(baseUrl, multiplier)
```
and resolve the baseUrl in the method that calls this.  End result is the same, it is just a bit strange to have a method like this require a lot more than it really needs to work.

Reply:
made change